Mathlib/Algebra/BigOperators/Basic.lean

```diff
@@ -119,14 +119,14 @@ macro_rules (kind := bigprod)
 
 /-- `∑ x in s, f x` is notation for `Finset.sum s f`. It is the sum of `f x`,
 where `x` ranges over the finite set `s`. -/
-syntax (name := bigsumin) "∑ " extBinder "in " term "," term : term
+syntax (name := bigsumin) "∑ " extBinder "in " term "," term:51 : term
```

Comment:
The library note around lines 84~100 above suggests that we use precedence level 67 everywhere. Should that also work in Lean 4?

Reply:
Good idea. Done.